L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -95,18 +99,30 @@ using namespace edm;
 //                          //
 //////////////////////////////
 
-class L1TrackObjectNtupleMaker : public edm::EDAnalyzer {
+class L1TrackObjectNtupleMaker : public edm::stream::EDAnalyzer<> {
```

Comment:
This module uses `TFileService` which is not thread safe, and therefore the base class must be
```suggestion
class L1TrackObjectNtupleMaker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
```
(plus the call to `usesResource(TFileService::kSharedResource);` in the constructor)